p_view.c

```diff
@@ -980,125 +980,140 @@ void ClientEndServerFrame (edict_t *ent)
 
 	current_player = ent;
 	current_client = ent->client;
```

Comment:
this file kinda got mangled because of spacing, but this is called in RunFrame to ensure HUD keeps working properly & players can check scores during pausing, so pausing needs to be careful here